cli/tests/test_alias.rs

```diff
@@ -41,20 +41,20 @@ fn test_alias_legacy_section() {
     let repo_path = test_env.env_root().join("repo");
 
     // Can define aliases in [alias] section
-    test_env.add_config(r#"alias.b = ["log", "-r", "@", "-T", "bookmarks"]"#);
+    test_env.add_config(r#"alias.bk = ["log", "-r", "@", "-T", "bookmarks"]"#);
     test_env.jj_cmd_ok(&repo_path, &["bookmark", "create", "my-bookmark"]);
-    let stdout = test_env.jj_cmd_success(&repo_path, &["b"]);
+    let stdout = test_env.jj_cmd_success(&repo_path, &["bk"]);
     insta::assert_snapshot!(stdout, @r###"
     @  my-bookmark
     │
     ~
     "###);
 
     // The same alias (name) in both [alias] and [aliases] sections is an error
-    test_env.add_config(r#"aliases.b = ["bookmark", "list"]"#);
-    let stderr = test_env.jj_cmd_failure(&repo_path, &["b"]);
+    test_env.add_config(r#"aliases.bk = ["bookmark", "list"]"#);
+    let stderr = test_env.jj_cmd_failure(&repo_path, &["bk"]);
     insta::assert_snapshot!(stderr, @r###"
-    Error: Alias "b" is defined in both [aliases] and [alias]
+    Error: Alias "bk" is defined in both [aliases] and [alias]
```

Comment:
Btw, these tests probably didn't have to change now that the alias is defined in the config file, but it still seems reasonable to use a different name here.